src/platform/packages/shared/kbn-esql-ast/src/commands_registry/options/recommended_queries/index.ts

```diff
@@ -201,6 +202,10 @@ export const getRecommendedQueriesSuggestionsFromStaticTemplates = async (
       kind: 'Issue',
       detail: query.description,
       sortText: query?.sortText ?? 'E',
+      category:
+        query.sortText === 'D'
+          ? SuggestionCategory.RECOMMENDED_QUERY_SEARCH
```

Comment:
Is this category tracking the extensions queries? Can we rename it to `CONTEXTUAL_RECOMMENDED_QUERY`